llama_stack/distribution/server/auth_providers.py

```diff
@@ -162,18 +177,99 @@ async def close(self):
             self._client = None
 
 
+JWT_AUDIENCE = "llama-stack"
```

Comment:
This should perhaps be configurable, in case you want to run distinct servers but use the same authorization service for all of them.